cmip6_downscaling/disagg/terraclimate.py

```diff
@@ -341,33 +367,34 @@ def hydromod(t_mean, ppt, pet, awc, soil, snow_storage, mfsnow):
         aet = pet
         deficit = 0
 
-    # this boolean is hardcoded true right now
-    if snow_drives_hydrology:
-        excess = max(0, soil + delta_soil - awc)
-        excess_rain_only = max(0, soil + excess_after_liquid - awc)
-        if (demand < supply) and (soil + delta_soil > awc):
-            # if supply exceeds demand and the updated soil water is greater than
-            # the available capacity
-            runoff = excess
-            runoff_snow += excess - excess_rain_only
-            soil = awc
-        elif (demand < supply) and (soil + delta_soil <= awc):
-            # if supply exceeds demand and updated soil water
-            # is less than available water capacity
-            soil += delta_soil
-            runoff = 0
+    excess = max(0, soil + delta_soil - awc)
+    excess_rain_only = max(0, soil + excess_after_liquid - awc)
+    if (demand < supply) and (soil + delta_soil > awc):
+        # if supply exceeds demand and the updated soil water is greater than
+        # the available capacity
+        runoff = excess
+        runoff_snow += excess - excess_rain_only
+        soil = awc
+    elif (demand < supply) and (soil + delta_soil <= awc):
+        # if supply exceeds demand and updated soil water
+        # is less than available water capacity
+        soil += delta_soil
+        runoff = 0
```

Comment:
@orianac - can this just be an if/else?

Its possible that we never set runoff here.